### PR TITLE
Add example/gh_traffic.py and needed documentation

### PR DESCRIPTION
**Pull request by mlangbehn**
example/gh_traffic.py is now available to demonstrate how to pull GitHub
repository traffic statistics via the GitHub REST API v3. Documentation has been updated
accordingly.

EDIT:
* Changed prompt for repo_path per @animeshsingh's comment
* Added variable for gh_api_url so that it would be easy to adapt
  example/gh_traffic.py for use with GitHub Enterprise

Signed-off-by: Matt Langbehn <matthew.langbehn@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,25 @@ WORK IN PROGRESS
...
+
+To use the [sample python script](example/gh_traffic.py), you will need to [create a personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) with public_repo access:
+![personal_access_token_settings](images/personal_access_token_settings.png "Personal Access Token Settings")
+
+Then, run [`example/gh_traffic.py`](example/gh_traffic.py), entering your personal access token and repo path when asked:
+
+```shell
+$ $(which python) example/gh_traffic.py
+GitHub personal access token: ****************************************
+GitHub repo (username/reponame): IBM/github-traffic-stats-collector
```
Comment (animeshsingh) [inline README.md]: In this case this organization_name/reponame. Maybe you can say anything after github.com, or have them enter the full URL

Comment (mlangbehn) [reply README.md]: [Replied in `example/gh_traffic.py`](https://github.com/IBM/github-traffic-stats-collector/pull/3#discussion_r131280285).

File: example/gh_traffic.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+"""
+An simple tool to pull GitHub Traffic Statics for a repository
+"""
+
+import json
+import urllib2
+
+auth_token = 'token ' + raw_input("GitHub personal access token: ")
+
+repo_path = raw_input("GitHub repo (username/reponame): ")
```
Comment (mlangbehn) [inline example/gh_traffic.py]: @animeshsingh: Replying to your comment in README.md here.

What if we change this line to this?:

```python
repo_path = raw_input("GitHub repo (username_or_orgname/reponame): ")
```

Comment (animeshsingh) [reply example/gh_traffic.py]: Makes sense

**Pull request merged**